Stripe/STPLegalEntityParams.h

```diff
@@ -0,0 +1,48 @@
+//
+//  STPLegalEntityParams.h
+//  Stripe
+//
+//  Created by Daniel Jackson on 12/20/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "STPFormEncodable.h"
+
+@class STPAddress;
+
+
+@interface STPVerificationParams: NSObject<STPFormEncodable>
+@property (nonatomic, copy, nullable) NSString *document;
+@end
+
+
+@interface STPPersonParams: NSObject<STPFormEncodable>
+@property (nonatomic, copy, nullable) NSString *firstName;
+@property (nonatomic, copy, nullable) NSString *lastName;
+@property (nonatomic, copy, nullable) NSString *maidenName;
+@property (nonatomic, strong, nullable) STPAddress *address;
+@property (nonatomic, copy, nullable) NSDateComponents *dateOfBirth;
+@property (nonatomic, strong, nullable) STPVerificationParams *verification;
+@end
+
+
+typedef NS_ENUM(NSUInteger, STPLegalEntityType) {
+    STPLegalEntityTypeIndividual,
+    STPLegalEntityTypeCompany,
+};
+
+
+@interface STPLegalEntityParams : STPPersonParams
```

Comment:
Do `STPVerificationParams` and `STPPersonParams` belong in their own files? Or are they fine here?

They're *only* used here (for now?), **and** the inheritance also links `STPLegalEntityParams` with `STPPersonParams` pretty closely.

Reply:
I think they're fine together!